src/string_bytes.h

```diff
@@ -27,7 +27,7 @@
 // Decodes a v8::Local<v8::String> or Buffer to a raw char*
 
 #include "v8.h"
-#include "env.h"
+#include "env-inl.h"
```

Comment:
Can you explain why this is necessary? If an inline function here uses an inline function from `env.h`, then it’s probably best to either make it non-inline and move it to the .cc file, or create a `string_bytes-inl.h` that defines that function and includes `env-inl.h`?

Reply:
The more I think of it, it's probably not necessary and an over reaction on my side to this CI error on osx https://ci.nodejs.org/job/node-test-commit-osx/29819/nodes=osx1011/console

Why would only that build fail for that particular reason still leaves me puzzled

Reply:
Yeah…  I don’t know either, but I think it makes sense to move `StringBytes::InlineDecoder::Decode()` into a `string_bytes-inl.h`?